eox_tagging/api/v1/filters.py

```diff
@@ -11,26 +11,26 @@
 class TagFilter(filters.FilterSet):
     """Filter class for tags."""
 
-    course_id = filters.CharFilter(field_name="course_id", method="filter_by_target_object")
-    username = filters.CharFilter(field_name="username", method="filter_by_target_object")
-    enrolled = filters.CharFilter(field_name="enrollment", method="filter_by_target_object")
+    course_id = filters.CharFilter(method="filter_by_target_object")
+    username = filters.CharFilter(method="filter_by_target_object")
+    enrolled_in = filters.CharFilter(method="filter_by_target_object")
     enrollments = filters.CharFilter(method="filter_enrollments")
     target_type = filters.CharFilter(method="filter_target_types")
-    created_at = filters.DateTimeFromToRangeFilter(field_name="created_at")
-    activation_date = filters.DateTimeFromToRangeFilter(field_name="activation_date")
+    created_at = filters.DateTimeFromToRangeFilter()
+    activation_date = filters.DateTimeFromToRangeFilter()
     access = filters.CharFilter(method="filter_access_type")
 
     class Meta:  # pylint: disable=old-style-class, useless-suppression
         """Meta class."""
         model = Tag
-        fields = ['key', 'created_at', 'activation_date', 'status', 'course_id', 'enrolled', 'enrollments', 'username']
+        fields = ['key', 'status']
```

Comment:
What is the consequence of removing this? Will the created_at, activation_date and so filters still work?

Reply:
None. 
`fields` option generates filters given the model fields, so, since we are defining explicitly filters for created_at, activation_date and so on they should work the same.
I'm currently testing all filters and it seems to be working.